transform/snowflake-dbt/models/finance/contracted_arr_reporting.sql

```diff
@@ -0,0 +1,161 @@
+{{config({
+    "materialized": "table",
+    "schema": "finance",
+    "tags":'nightly'
+  })
+}}
+
+--v1 Roy CONTRACTED ARR REPORTING
+--tested on snowflake 20230606
+--this sql model reports ARR on date contracted instead of report date where report date is the later of close date or license start
```

Comment:
Is "date contracted" same as "date closed" (comments are slightly different in the two files).

Reply:
Thanks, so for clarity, should the comments refer to/use the same terminology in the two files? Like "this sql model reports ARR on date contracted instead of report date" (this file) vs. "Same as arr_reporting but reporting date shifts to closing date instead of reporting date" (schema.yml)

Reply:
Yes

Reply:
@catalintomai it would be helpful if you directly ask the concern you are addressinf

Reply:
@RoyI99 , it is more of a prevention of future/possible issues for me. If in the future someone new to MM and not familiar with this terminology will have to understand the logic here, they might ask why is the description in the schema file slightly different from the description of top of the file the schema file entry references (like the original q: is "date contracted" same as "closing date").

Reply:
Yes zoom call is better

Reply:
@catalintomai I pushed a second committ to clarify on the heading the difference between contract and close day.  Will document the rest on confluence.  Please approve so this will be part of the refresh tonight.   Thanks again!